app/models/messaging_preference.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+require 'common/models/base'
+# Secure Messaging Notification Preference Model
+class MessagingPreference < Common::Base
+  FREQUENCY_UPDATE_MAP = {
+    'none' => 0,
+    'each_message' => 1,
+    'daily' => 2
+  }.freeze
+
+  FREQUENCY_GET_MAP = {
```

Comment:
Hmm, my only concern would be if MHV changes these and doesn't tell us. Because technically we're not following the intended means of verifying with that endpoint. Maybe we can put a TODO comment in there and revisit it once we build out that caching layer that is backlogged. If it is cached (outside the user session), then any user requesting that endpoint would fetch the cached value from redis until the cache expires. I think 24 hour or more for the cache seems reasonable there.